Tests/StringsTest.php

```diff
@@ -13,14 +13,14 @@
 
 class StringsTest extends TestCase {
 
-    public function providerEncodings() {
+    public static function providerEncodings() {
         return array(
             array('UTF-8'),
             array('ISO-8859-1'),
         );
     }
 
-    public function providerSnakeAndCamelCase() {
+    public static function providerSnakeAndCamelCase() {
```

Comment:
Why did we need to change this methods to static? It was because phpunit 10 needed them to be that way? If we are switching to phpunit 9.6, I think we can keep them to be non-static for now.